packages/plugins/i18n/server/constants/iso-locales.json

```
@@ -1095,6 +1095,10 @@
     "code":"gl-ES",
     "name":"Galician (Spain) (gl-ES)"
   },
+  {
+    "code":"lao",
```

```suggestion
    "code":"lo",
```
It looks like we do use 3-letter codes elsewhere, but do we have any documented reason why sometimes we use 2-letter and sometimes 3?

---

We're following ISO denomination
https://en.wikipedia.org/wiki/ISO_639-3

---

But I do agree it's confusing because sometimes we used ISO_639_2 & in other cases ISO_639-3.
It depends heavily on the locale, but basically, ISO_639_3 has more granularity

---

Yeah, I'm just confused by the inconsistency. I suppose it's fine since the damage is already done, I'll look at it as "start shifting towards 3-letter with anything new"